docs/src/modules/components/AppSearch.js

```diff
@@ -174,150 +71,425 @@ const Shortcut = styled('div')(({ theme }) => {
   };
 });
 
-/**
- * When using this component it is recommend to include a preload link
- * `<link rel="preload" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css" as="style" />`
- * to potentially reduce load times
- */
+const NewStartScreen = () => {
+  const startScreenOptions = [
+    {
+      category: {
+        name: 'Getting Started',
+        icon: <ArticleOutlinedIcon className="DocSearch-NewStartScreenTitleIcon" />,
+      },
+      items: [
+        { name: 'Installation', href: '/getting-started/installation/' },
+        { name: 'Usage', href: '/getting-started/usage/' },
+        { name: 'Learn', href: '/getting-started/learn/' },
+      ],
+    },
+    {
+      category: {
+        name: 'Popular Searches',
+        icon: <ToggleOffOutlinedIcon className="DocSearch-NewStartScreenTitleIcon" />,
+      },
+      items: [
+        { name: 'Material Icons', href: '/components/material-icons/' },
+        { name: 'Text Fields', href: '/components/text-fields/' },
+        { name: 'Button', href: '/components/buttons' },
+      ],
+    },
+    {
+      category: {
+        name: 'Customization',
+        icon: <CreateOutlinedIcon className="DocSearch-NewStartScreenTitleIcon" />,
+      },
+      items: [
+        { name: 'How To Customize', href: '/customization/how-to-customize/' },
+        { name: 'Theming', href: '/customization/theming/' },
+        { name: 'Default Theme', href: '/customization/default-theme/' },
+      ],
+    },
+    {
+      category: {
+        name: 'System',
+        icon: <BuildOutlinedIcon className="DocSearch-NewStartScreenTitleIcon" />,
+      },
+      items: [
+        { name: 'Basics', href: '/system/basics/' },
+        { name: 'Properties', href: '/system/properties/' },
+        { name: 'The sx prop', href: '/system/the-sx-prop/' },
+      ],
+    },
+  ];
+  return (
+    <div className="DocSearch-NewStartScreen">
+      {startScreenOptions.map(({ category, items }) => (
+        <div key={category.name} className="DocSearch-NewStartScreenCategory">
+          <div className="DocSearch-NewStartScreenTitle">
+            {category.icon}
+            {category.name}
+          </div>
+          {items.map(({ name, href }) => (
+            <Link key={name} className="DocSearch-NewStartScreenItem" href={href}>
+              {name}
+              <KeyboardArrowRightRounded className="DocSearch-NewStartScreenItemIcon" />
+            </Link>
+          ))}
+        </div>
+      ))}
+    </div>
+  );
+};
+
 export default function AppSearch() {
-  const inputRef = React.useRef(null);
-  const [focused, setFocused] = React.useState(false);
-  const theme = useTheme();
+  useLazyCSS(
+    'https://cdn.jsdelivr.net/npm/@docsearch/css@3.0.0-alpha.40/dist/style.min.css',
```

Comment:
Do you know why `@docsearch/css` is a dependency of `@docsearch/react`? Maybe we don't need this hook anymore.

Reply:
They just reexport from `@docsearch/react` [in here](https://github.com/algolia/docsearch/blob/next/packages/docsearch-react/style/index.js) but I don't see it is using in their component.

What if we import the module like this? so that if the package version updated we don't need to update multiple places.
```js
import "@docsearch/react/style" // equal to @docsearch/css
```

Reply:
> What if we import the module like this? so that if the package version updated we don't need to update multiple places.

I am trying this

Reply:
![image](https://user-images.githubusercontent.com/4512430/134295479-0f3c2a97-726a-4c80-b8b3-0daeff018850.png)

Reply:
We need to keep this.

Reply:
We can go with the current approach and open another PR for improvement?